### ISSUE TITLE
Too many links in the Remarks section

### ISSUE THREAD

**Issue by phunanon** [dotnet/dotnet-api-docs#1192]
I suggest the links to other topics appear only once, at the first mention.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a38fc498-25ab-28d3-bce0-ce91949f358b
* Version Independent ID: 90974c26-6c6f-c367-1442-7ff1e7c6b244
* Content: [TcpClient.Client Property (System.Net.Sockets)](https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.tcpclient.client?view=netframework-4.7.2#feedback)
* Content Source: [xml/System.Net.Sockets/TcpClient.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Net.Sockets/TcpClient.xml)
* Product: **dotnet-api**
* GitHub Login: @karelz
* Microsoft Alias: **ncldev**

**Comment by karelz** [comment]
I assume you have `Remarks` section in mind.
I think that the general docs rule is to hyperlink all references to classes, members, etc.

cc @mairaw

**Comment by mairaw** [comment]
I agree that are too many links. I'll take care of this particular occurrence but there might be many more like this in our docs.
